mix.exs

```diff
@@ -45,10 +45,18 @@ defmodule ExMicrosoftbot.Mixfile do
         disable_token_validation: false
```

Comment:
Increase the version of the package by incrementing major version "3.0.0" as it contains some breaking changes.